lambda/docker-runner.yaml

```diff
@@ -0,0 +1,111 @@
+# This template depends on having an AWS EFS in place which can be deployed with
+# https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/EFS/efs.yaml
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: S3Objects
+Description: >-
+  Docker runner product
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+          default: Container Image
+        Parameters:
+          - ImageUri
+          - EntryPoint
+          - Command
+          - WorkingDirectory
+          - MemorySize
+      - Label:
+          default: Synapse
+        Parameters:
+          - SynapseUser
+          - SynapsePassword
+      - Label:
+          default: Execution Trigger
+        Parameters:
+          - EnableSchedule
+          - Schedule
+Parameters:
+  MemorySize:
+    Type: Number
+    MinValue: 128
+    MaxValue: 10240
+    Default: 128
```

Comment:
This default is probably too low... but arriving at the right default is probably trial and error

Reply:
please make a suggestion

Reply:
An EC2 t3.micro uses 1 GB so why not make this the same (1024 MB)?

Reply:
done